### DISCUSSION TITLE
Do my drives need to be connected?

### DISCUSSION THREAD

**Discussion by ihavenoclueofwhatimdoing** [Chia-Network/chia-blockchain#3592] [Q&A]
I'm going through the Chia Wiki to get an understanding, but what I don't get is how the "winner" is selected. Let's say I've 10TB plotted on a drive and after everything is done, I unplug it. Does it mean, that I my unplugged drive is no longer considered when the winner is chosen, or do I get the price once I reconnect my drive? Is that what is meant by proof of time? So if I'm not connecting my drive, that means I lose the chance on getting the coin? - I'm sorry, I literally have no clue of what I'm doing

**Comment by Muffexx** [comment]
Hello @ihavenoclueofwhatimdoing,

all the plots need to be accessible at all times in order to enable looking up qualities (random reads) if one of your plots is eligible. So the drives must stay connected for you to have a chance to win.

regards
muffexx
